### PR TITLE
Enh/pushvars

### PR DESCRIPTION
**Pull request by guillep**
[Add pushLiteralVariable and pushTemporary in druid jit](https://github.com/pharo-project/pharo-vm/commit/5a16f2fca06d73548075033fd310c44b430d5189)

### CODE REVIEW

File: smalltalksrc/VMMaker/DruidJIT.class.st [hunk 1]
```diff
@@ -26,46 +26,262 @@ DruidJIT class >> bytecodeTable [
...
+		  { 1. 23. 23. #unknownBytecode }.
+		  { 1. 24. 24. #unknownBytecode }.
+		  { 1. 25. 25. #unknownBytecode }.
+		  { 1. 26. 26. #unknownBytecode }.
+		  { 1. 27. 27. #unknownBytecode }.
+		  { 1. 28. 28. #unknownBytecode }.
+		  { 1. 29. 29. #unknownBytecode }.
+		  { 1. 30. 30. #unknownBytecode }.
+		  { 1. 31. 31. #unknownBytecode }.
+		  { 1. 32. 32. #gen_PushLiteralConstantBytecode0 }.
```
Comment (PalumboN) [inline smalltalksrc/VMMaker/DruidJIT.class.st:61]: We can generate all of these. Or do you want to measure only the list?

**Review by PalumboN — COMMENTED** [review]
CI is looping 🔁

File: smalltalksrc/VMMaker/DruidJIT.class.st (hunk 1 above)
Comment (guillep) [reply smalltalksrc/VMMaker/DruidJIT.class.st:61]: Oh yes, I go by the list :D

Comment (guillep) [reply smalltalksrc/VMMaker/DruidJIT.class.st:61]: I know we can, we are super great ;) but I'm a maniac

**Pull request merged**